scripts/common/host-packages.sh

```diff
@@ -279,10 +279,18 @@ gpgcheck=0
 repo_gpgcheck=0
 EOF
 
-    if [[ "${packages[*]}" == *"containerd.io"* ]] ; then
-        yum install --allowerasing -y "${packages[@]}"
+    if [ -n "$KURL_USE_RHEL9_REMOTE_YUM_REPOS" ] ; then
+        if [[ "${packages[*]}" == *"containerd.io"* ]] ; then
+            yum install --allowerasing -y "${packages[@]}"
+        else
+            yum install -y "${packages[@]}"
+        fi
     else
-        yum install -y "${packages[@]}"
+        if [[ "${packages[*]}" == *"containerd.io"* ]] ; then
+            yum install --disablerepo=* --enablerepo="$reponame" --allowerasing -y "${packages[@]}"
+        else
+            yum install --disablerepo=* --enablerepo="$reponame" -y "${packages[@]}"
+        fi
     fi
```

Comment:
Instead of adding this if block, can we pass an env var that either includes the `--disablerepo=*` flag or not?

Reply:
Let me know if this is what you meant.